packages/expo-screen-orientation/ios/ScreenOrientationUtilities.swift

```diff
@@ -73,6 +73,23 @@ internal func plistStringToInterfaceOrientationMask(_ maskName: String) -> UIInt
   }
 }
 
+extension UIInterfaceOrientationMask.Element {
+  internal init?(fromOrientationString orientationString: String) {
```

Comment:
Can you make it a function instead? It does seem weird to extend an associated type – in case of this mask, `Element` is probably a `UInt`, but you have to guess that, otherwise it is not clear right away.

Also, adding an initializer through the extension is something we should avoid in my opinion. Initializers are strongly associated with the object-oriented model, but extensions are not for that (you can't extend another class with the extension, you can't override existing functions etc.).

Moreover, in Swift there are two types of initializers – designated and convenience. This one looks like a convenience initializer, but it's not marked with the `convenience` keyword and doesn't call a designated initializer, so it gets confusing.